lib/scrabble.rb

```diff
@@ -1,13 +1,53 @@
 class Scrabble
-  def point_values
-    {
+  attr_reader     :point_values,
+                  :counter
+
+  def initialize
+    @point_values = {
       "A"=>1, "B"=>3, "C"=>3, "D"=>2,
       "E"=>1, "F"=>4, "G"=>2, "H"=>4,
       "I"=>1, "J"=>8, "K"=>5, "L"=>1,
       "M"=>3, "N"=>1, "O"=>1, "P"=>3,
       "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
+      "Y"=>4, "Z"=>10, " "=>0, nil=>0
     }
+    @counter      = []
+  end
+
+  def score(word)
+    if word == nil
+      return 0
+    end
+    new_word = word.upcase
+    new_word.chars.each do |letter|
+      @counter << @point_values[letter]
+    end
+    new_count = @counter.inject(0){|sum, num| sum + num}
```

Comment:
This is a great use of inject, but you don't need to assign this to a variable, you could just have the inject as the last 'line' of code.